workshop/workshop-logistics.md

```diff
@@ -12,11 +12,23 @@ nav_title: Logistics
     {% include_relative participant-information.md %}
 {% endif %}
 
+<!-- Define workshop content based on `site.workshop_content` -->
+
+{% case site.workshop_content  %}
+{% when "intro-r" %}
+  {% assign workshop_content = "the R programming language" %}
```

Comment:
Question - Do we care that this variable is highly similar to the `site.` variable? Maybe `workshop_content_string` would be better?

Reply:
That is what scope is for, so I wouldn't worry about it.


You could also just inline this case statement and avoid the variable entirely. It is less pretty, but maybe allows for easier editing later, since the sentence context is right there.

Reply:
I'd probably inline it, but it isn't a very strong preference.